stdlib/contrib/anaisdg/anomalydetection/mad.flux

```diff
@@ -0,0 +1,40 @@
+package anomalydetection 
+
+import "math"
+import "experimental"
+
+mad = (table=<-, threshold) => {
```

Comment:
Should users always have to specify the `threshold` or is there a reasonable default?

Reply:
There is a reasonable default. threshold = 3.0. Thank you.